src/Core/Varargs/PrintfFormatParser.cs

```diff
@@ -18,16 +18,13 @@
  */
 #endregion
```

Comment:
The Microsoft `wsprintf` function obeys a slightly different syntax from standard C `printf`. I don't have the exact difference at hand, but it makes sense for `Windows.MsPrintfFormatParser` to inherit from `Core.Analysis.PrintfFormatParser`.